Terminal.Gui/Views/TextView.cs

```diff
@@ -745,6 +898,40 @@ void ProcMovePrev (ref int nCol, ref int nRow, Rune nRune)
 				return null;
 			}
 		}
+
+		/// <summary>
+		/// Converts the string into a <see cref="List{RuneCell}"/>.
+		/// </summary>
+		/// <remarks>
+		/// This is a Terminal.Gui extension method to <see cref="string"/> to support TUI text and attribute manipulation.
```

Comment:
Delete this line.

Reply:
Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/0f173e0a83002efc68274863c49b1001e934c176.